### PR TITLE
migrates page_token to redis

### PR DESCRIPTION
**Pull request by Faiz**
## What does this PR do?
this does what told me to do. the redis thing

## ✅ Pull Request checklist

- [ ] Is this code complete?
- [ ] Are tests done/modified?

## 📃 Link to PR updating documentation (if applicable)

https://github.com/uclapi/apiDocs/pulls

## 🚨 Is this a breaking change for API clients?
Yes/No

## :squirrel: Deploy notes
For example: Need to run migrations as part of deployment

### CODE REVIEW

**Comment by ChristopherHammond13** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://jenkins.uclapi.com/job/UCLAPI/30/
Jenkins statics successfully built.

File: backend/uclapi/roombookings/views.py [hunk 1]
```diff
@@ -50,7 +50,7 @@ def get_rooms(request, *args, **kwargs):
 
 
 @api_view(['GET'])
-@uclapi_protected_endpoint()
+# @uclapi_protected_endpoint()
```
Comment (ChristopherHammond13) [inline backend/uclapi/roombookings/views.py]: Debug?

**Comment by Faiz** [comment]
Done. Get it merged. Make sure to delete that table from default.

**Comment by ChristopherHammond13** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://jenkins.uclapi.com/job/UCLAPI/33/
Jenkins statics successfully built.

**Comment by ChristopherHammond13** [comment]
Good stuff. Now that page tokens aren't necessary shall we remove the model then do a `makemigrations` and `migrate`?

**Comment by Faiz** [comment]
yep. I will do that commit now

**Comment by ChristopherHammond13** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://jenkins.uclapi.com/job/UCLAPI/34/
Jenkins statics successfully built.

**Comment by ChristopherHammond13** [comment]
Almost ready to go I'd say! Just quick one: should we reset the `ex=` parameter back to `30*60` every time the page token is used, or forcefully expire it after the time expires?

**Comment by ChristopherHammond13** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://jenkins.uclapi.com/job/UCLAPI/36/
Jenkins statics successfully built.

**Comment by ChristopherHammond13** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://jenkins.uclapi.com/job/UCLAPI/37/
Jenkins statics successfully built.

**Pull request merged**